test_last_requests.py

```diff
@@ -0,0 +1,16 @@
+import httpretty
+import requests
+
+
+def test_get_all_requests():
```

Comment:
you should have used `@httpretty.activate` instead of manually enabling and disabling, so that if your test fails it will still disable httpretty without causing side effects.
Also it would have been one less line of code :)